src/shared/extensions/redhat-yaml/schema-extension-api.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// sourced from https://github.com/redhat-developer/vscode-yaml/blob/3d82d61ea63d3e3a9848fe6b432f8f1f452c1bec/src/schema-extension-api.ts
+// removed everything that is not currently being used
+
+export interface ExtensionAPI {
```

Comment:
nit: is this required to be in a separate file? Would prefer to put everything in `shared/extensions/redhat-yaml.ts` , rather than spread things over multiple files. We are unlikely to have lots of code wrapping a particular extension.

Reply:
Yeah it should be with the implementation. The file was larger initially so it made sense to keep it separate at the start.